recipes-openxt/xenclient/xenclient-toolstack_git.bb

```diff
@@ -1,8 +1,9 @@
 DESCRIPTION = "XenClient toolstack"
 LICENSE = "LGPLv2.1"
 LIC_FILES_CHKSUM = "file://COPYING;md5=321bf41f280cf805086dd5a720b37785"
-DEPENDS += "ocaml-cross ocaml-dbus ocaml-camomile xen xen-libxl xz"
+DEPENDS += "ocaml-cross ocaml-dbus ocaml-camomile xen xz"
 RDEPENDS_${PN} = "xen-xenstore xen-xenstored"
+RDEPENDS_${PN}_xenclient-dom0 += "xen-xl"
```

Comment:
Apparently there is no longer a dependency from xenclient-toolstack to xen-xl, I can only see RDEPENDS on xen-xenstore from the scripts.
Why is there an RDEPENDS on xen-xl ?